From 6e6f302a5bfa72b195021c0053aab3286d12f87a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Kristian=20H=C3=B8gsberg?= <krh@bitplanet.net>
Date: Fri, 13 Sep 2013 14:53:16 -0700
Subject: [PATCH 25/38] os: Don't include xorg-server.h

Just always compile in ListenOnOpenFD and AddClientOnOpenFD, they
don't depend on anything from Quartz or Wayland.
---
 os/connection.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/os/connection.c b/os/connection.c
index a95e7da..0a858b7 100644
--- a/os/connection.c
+++ b/os/connection.c
@@ -64,7 +64,6 @@ SOFTWARE.
 #include <dix-config.h>
 #endif
 
-#include <xorg-server.h>
 #ifdef WIN32
 #include <X11/Xwinsock.h>
 #endif
@@ -1259,7 +1258,6 @@ MakeClientGrabPervious(ClientPtr client)
     }
 }
 
-#if defined(XQUARTZ) || defined(XORG_WAYLAND)
 /* Add a fd (from launchd) to our listeners */
 void
 ListenOnOpenFD(int fd, int noxauth)
@@ -1334,5 +1332,3 @@ AddClientOnOpenFD(int fd)
         return;
     }
 }
-
-#endif
-- 
1.8.4.2

